docs/.vuepress/config.js

```diff
@@ -130,6 +130,7 @@ module.exports = {
         title: 'API'
       },
       '/samples/': [
+        'information',
```

Comment:
Maybe it should be index, just to make it obvious it's the start page.
Could also remove the redirect in that case

Reply:
It feels indeed more obvious as the start page but I can't seem to manage to also get it in the sidebar so you can click back to it. So you will need to click the `samples` button in the head again which makes it bit more confusing if you want to go back to it I think.

I changed filename to index.md and tried adding nothing in the `/samples/` array which gives the above described behaviour.

When I try adding `'index'` as the first item or an empty string as done for the normal docs part it doesnt update the sidebar and still shows the one from the normal docs

Reply:
Ok, information should be ok in that case.